api/src/api/userRouter.ts

```diff
@@ -76,11 +79,31 @@ const businessHasFormed = (userData: UserData): boolean => {
   return userData.formationData.getFilingResponse?.success ?? false;
 };
 
+const handleTaxId = async (
+  userData: UserData,
+  taxEncryptionDecryptionFactory: EncryptionDecryptionClient
+): Promise<UserData> => {
```

Comment:
there's no reason for this function to exist outside the context of the userRouter. if it was contained within the router, then you would not have to pass the encryption client in as an argument

Reply:
also, let's be careful about naming here. is that variable really a taxEncryptionDecryption_Factory_?

remember, a factory is a function that _returns_ an object